nix/package.nix

```diff
@@ -0,0 +1,6 @@
+{ pkgs, buildGoModule }:
+buildGoModule {
+  name = "mpd-mpris";
+  src = ./..;
+  vendorSha256 = "sha256-GmdD/4VYp3KeblNGgltFWHdOnK5qsBa2ygIYOBrH+b0=";
```

Comment:
I'd recommend `gomod2nix` so you won't have to maintain this hash, and instead run `gomod2nix` everytime you `go get -u`.